### ISSUE TITLE
[self-update] Problem with command, after update.

### ISSUE THREAD

**Issue by sargath** [hechoendrupal/DrupalConsole#1441]
After todays update I'm getting 

```
sargath ~/public_html $ drupal self-update

                                                                                                                        
 [WARNING] The latest version 0.9.9, was already installed on your system.                                              
                                                                                                                        
PHP Catchable fatal error:  Argument 1 passed to Symfony\Component\Console\Application::setDispatcher() must implement interface Symfony\Component\EventDispatcher\EventDispatcherInterface, null given, called in phar:///home/sargath/PHP-Tools/drupal/src/Command/SelfUpdateCommand.php on line 63 and defined in phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Application.php on line 92
PHP Stack trace:
PHP   1. {main}() /home/sargath/PHP-Tools/drupal:0
PHP   2. require() /home/sargath/PHP-Tools/drupal:10
PHP   3. require() phar:///home/sargath/PHP-Tools/drupal/bin/console:3
PHP   4. Symfony\Component\Console\Application->run() phar:///home/sargath/PHP-Tools/drupal/bin/console.php:83
PHP   5. Drupal\Console\Application->doRun() phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Application.php:120
PHP   6. Symfony\Component\Console\Application->doRun() phar:///home/sargath/PHP-Tools/drupal/src/Application.php:224
PHP   7. Symfony\Component\Console\Application->doRunCommand() phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Application.php:189
PHP   8. Symfony\Component\Console\Command\Command->run() phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Application.php:846
PHP   9. Drupal\Console\Command\SelfUpdateCommand->execute() phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Command/Command.php:256
PHP  10. Symfony\Component\Console\Application->setDispatcher() phar:///home/sargath/PHP-Tools/drupal/src/Command/SelfUpdateCommand.php:63

Catchable fatal error: Argument 1 passed to Symfony\Component\Console\Application::setDispatcher() must implement interface Symfony\Component\EventDispatcher\EventDispatcherInterface, null given, called in phar:///home/sargath/PHP-Tools/drupal/src/Command/SelfUpdateCommand.php on line 63 and defined in phar:///home/sargath/PHP-Tools/drupal/vendor/symfony/console/Application.php on line 92
```

**Comment by jmolivas** [comment]
@sargath: Thanks for reporting.

**Comment by geerlingguy** [comment]
See also: https://github.com/geerlingguy/drupal-vm/issues/334

Many people are reporting issues with Drupal Console preventing Drupal VM configuration today.

**Comment by sargath** [comment]
@jmolivas n/p, @geerlingguy nope, im running on Archlinux as a main sys, without any automatization/vagrant/ansible etc

**Comment by geerlingguy** [comment]
@sargath - I'm not saying it's related to automation, just that I'm getting the same error :)

**Comment by jmolivas** [comment]
Fixed with #1453

**Comment by jmolivas** [comment]
Fix will be included on the v0.9.10 release.
